docs/windows.md

```diff
@@ -1,142 +1,141 @@
 ```
 
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.
+
+### `gx`
+You may install prebuilt binaries for [`gx`](https://dist.ipfs.io/#gx) & [`gx-go`](https://dist.ipfs.io/#gx-go) if they're available for your platform.
+Alternatively you can build them from source and install them to `%GOPATH%\bin` by running the following:
 
 ```
-go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=XXXXXXX"
+go get -u github.com/whyrusleeping/gx
+go get -u github.com/whyrusleeping/gx-go
 ```
 
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
+### `go-ipfs`
 
 ```
-ipfs version --commit
+SET PATH=%PATH%;%GOPATH%\bin
+go get -u -d github.com/ipfs/go-ipfs
+cd %GOPATH%/src/github.com/ipfs/go-ipfs
+gx --verbose install --global
+cd cmd\ipfs
 ```
+We need the `git` commit hash to be included in our build so that in the extremely rare event a bug is found, we have a reference point later for tracking them. We'll ask `git` for it and store it in a variable. Unfortunately the syntax for the next command is different depending on how it's executed. If you're on the interactive command line vs executing from inside the batch/cmd interpreter. Use the one that applies to you.  
+interactive: `FOR /F %V IN ('git rev-parse --short HEAD') do set SHA=%V`  
+interpreter: `FOR /F %%V IN ('git rev-parse --short HEAD') do set SHA=%%V`  
 
-It should output something like "ipfs version 0.4.0-dev-XXXXXXX" where
-XXXXXXX is the current commit that you passed to the build command.
-
-## Cygwin way: install Cygwin
-
-Install Cygwin as explained in the Cygwin documentation:
-
-http://cygwin.com/install.html
-
-By default Cygwin will not install 'make', so you should click on the
-"Devel" category during the Cygwin installation process and then check
-the 'make' package.
-
-## Cygwin way: build go-ipfs
-
-To build go-ipfs using Cygwin you just need to open a Cygwin Terminal
-and then type the following commands:
-
+Finally, we'll build and test `ipfs` itself.
 ```
-cd $GOPATH/src/github.com/ipfs/go-ipfs
-make install
+go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=%SHA%"
+%GOPATH%\bin\ipfs.exe version --all
 ```
+You can check that the ipfs output versions match with `go version` and `git rev-parse --short HEAD`, if `ipfs.exe` executes and everything matches, then building was successful.
 
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
+## Troubleshooting
+- Git auth  
```

Comment:
Not a big deal, but I feel like these bullet titles might read better if formatted like:

```
- **Git auth:** If you get authentication problems…

- **Anything else:** Please search…
```